polars/polars-time/src/windows/duration.rs

```diff
@@ -655,11 +651,13 @@ fn new_datetime(
     min: u32,
     sec: u32,
     nano: u32,
-) -> NaiveDateTime {
-    let date = NaiveDate::from_ymd_opt(year, month, days).unwrap();
-    let time = NaiveTime::from_hms_nano_opt(hour, min, sec, nano).unwrap();
-
-    NaiveDateTime::new(date, time)
+) -> Option<NaiveDateTime> {
+    let date = NaiveDate::from_ymd_opt(year, month, days);
```

Comment:
You can simply use question mark to unwrap the `Some` elegantly and fast return the `None` case.